modules/samples/producers/http-performance/src/main/java/org/wso2/carbon/sample/http/performance/Http.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.sample.http.performance;
+
+import org.apache.http.client.HttpClient;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.SystemDefaultHttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.axiom.om.util.Base64;
+import org.apache.log4j.Logger;
+import com.google.gson.JsonObject;
+
+import java.text.DecimalFormat;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+
+public class Http implements Runnable {
+	private static Logger log = Logger.getLogger(Http.class);
+
+	static String url;
+	static String username;
+	static String password;
+	static long noOfEvents;
+	long count;
+	static int elapsedCount;
+	static int noOfPublishers;
+
+	public Http(){
+		count = 0;
+	}
+
+	public static void main(String args[]) {
+
+		log.info("Starting WSO2 Http Client");
+		url = args[0];
+		username = args[1];
+		password = args[2];
+		try{
+			noOfEvents = Long.parseLong(args[3]);
+			elapsedCount = Integer.parseInt(args[4]);
+			noOfPublishers = Integer.parseInt(args[5]);
+
+			ExecutorService executor = Executors.newFixedThreadPool(noOfPublishers);
+			for (int i = 0; i < noOfPublishers; i++) {
+				Runnable publisher = new Http();
+				executor.execute(publisher);
```

Comment:
Thanks for pointing it out @dilini-muthumala , change is done in pull request - https://github.com/wso2/product-cep/pull/160